viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if title and genre and rating:
+        new_movie = {}
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+
+    user_data["watched"].append(movie)
+    return user_data
+    
+def add_to_watchlist(user_data, movie):
+
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie_title):
+    for index in range(0, len(user_data["watchlist"])):
+        nested_movie_title_watched = user_data["watchlist"][index]["title"]
+        if movie_title in nested_movie_title_watched:
+            correct_movie = user_data["watchlist"][index]
+            user_data["watched"].append(correct_movie)
+            user_data["watchlist"].remove(correct_movie)

```

Comment:
This works just fine, but there's a lot going on! And I think we can make the for loop a bit more readable. By using `for movie in user_data["watchlist"]` is much more descriptive and shorter. We could also use a `break` at the end so that we aren't running the for loop more time than necessary

```suggestion
    for movie in user_data["watchlist"]:
        if movie_title == movie["title"]:
            user_data["watched"].append(movie)
            user_data["watchlist"].remove(movie)
```

 Another cool thing we can do is combine the last two lines because remove returns the item itself:
 `user_data["watched"].append(user_data["watchlist"].remove(movie))`